### PR TITLE
Update catalog mainly for Autoupgrade module

### PR DESCRIPTION
**Pull request by jolelievre**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.0.x
| Description?      | Update catalog mainly for Autoupgrade module
| Type?             | improvement
| Category?         | LO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | ~
| Related PRs       | ~
| How to test?      | ~
| Possible impacts? | ~

### CODE REVIEW

File: translations/default/AdminDesignHelp.xlf [hunk 1]
```diff
@@ -218,7 +218,8 @@
       <trans-unit id="cabfe7563d12c3e047000dc5703d9709">
         <source>Add new page category</source>
         <target>Add new page category</target>
-        <note>Line: 31</note>
+        <note>Line: 31
+Comment: Stable feature flags form</note>
```
Comment (atomiix) [inline translations/default/AdminDesignHelp.xlf:222]: The comment feels weird here 🤔

File: translations/default/AdminInternationalFeature.xlf [hunk 2]
```diff
@@ -859,7 +859,8 @@
       <trans-unit id="076b68505282c6c0654708db343d6673">
         <source>Add new currency</source>
         <target>Add new currency</target>
-        <note>Line: 31</note>
+        <note>Line: 31
+Comment: Stable feature flags form</note>
```
Comment (atomiix) [inline translations/default/AdminInternationalFeature.xlf:863]: Same comment here 😅

**Comment by eternoendless** [comment]
I'm not sure it's a good idea to add the autoupgrade, which is not a native module, in the core's catalog.

The wordings should be embedded in the module itself, using xlf files. They will work with 1.7.8+.

File: translations/default/AdminDesignHelp.xlf (hunk 1 above)
Comment (eternoendless) [reply translations/default/AdminDesignHelp.xlf:222]: That's a feature from TranslationToolBundle, it picks any comment around the line where the wording is picked up and adds it. This particular comment is weird though.

**Review by matks — APPROVED** [review]

**Pull request closed without merging**